### DISCUSSION TITLE
Documentation Request

### DISCUSSION THREAD

**Discussion by minhchu** [chakra-ui/panda#771] [Q&A] [answered]
Hi, I want to file a Documentation Request issue but it redirects to https://github.com/chakra-ui/panda/issues/new/choose (sort of a loop).

Btw, there seems to be an issue with the example in https://panda-css.com/docs/concepts/recipes#typescript-guide

```tsx
import { cva, type RecipeVariantProps } from '../styled-system/css'
 
const buttonStyle = cva({
  base: {
    color: 'red',
    textAlign: 'center'
  },
  variants: {
    size: {
      small: {
        fontSize: '1rem'
      },
      large: {
        fontSize: '2rem'
      }
    }
  }
})
 
export type ButtonVariants = RecipeVariantProps<typeof buttonStyle> // { size?: 'small' | 'large' }
 
export const Button = panda('button', buttonStyle)
```

In the last line, there is a `panda` function but I don't know where to import it from. May `panda` be `styled` imported from `styled-system/jsx`

**Comment by anubra266** [comment, marked as answer]
You're right @minhchu it should be `styled`

https://github.com/chakra-ui/panda/pull/772/files

**Comment by minhchu** [comment]
@anubra266 I see that you also fix the issue template. Thank you :100:
